modules/virt-delete-vm-web.adoc

```diff
@@ -1,22 +1,33 @@
 // Module included in the following assemblies:
 //
-// * virt/virtual_machines/virt-delete-vms.adoc
+// * virt/managing-vms/virt-delete-vms.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="virt-delete-vm-web_{context}"]
 
 = Deleting a virtual machine using the web console
 
-Deleting a virtual machine permanently removes it from the cluster.
+Deleting a virtual machine (VM) permanently removes it from the cluster.
+
+If the VM is delete protected, the *Delete* action is disabled in the VM's *Actions* menu. To delete the VM you must first disable the VM's delete protection setting.
+
+
 
 .Procedure
 
-. In the {product-title} console, click *Virtualization* -> *VirtualMachines* from the side menu.
+. From the {product-title} web console, choose your view:
 
-. Click the Options menu {kebab} beside a virtual machine and select *Delete*.
+    * For a virtualization-focused view, select *Administrator* → *Virtualization* → *VirtualMachines*.
+
+    * For a general view, navigate to *Virtualization* → *VirtualMachines*.
+
+. Click the *Actions* menu {kebab} beside a VM and select *Delete*.
 +
-Alternatively, click the virtual machine name to open the *VirtualMachine details* page and click *Actions* -> *Delete*.
+Alternatively, click the VM's name to open the *VirtualMachine details* page and click *Actions* -> *Delete*.
 
 . Optional: Select *With grace period* or clear *Delete disks*.
 
-. Click *Delete* to permanently delete the virtual machine.
+. Click *Delete* to permanently delete the VM.
+
+.Additional resources
+* link:https://openshift-enterprise/latest/virt/managing_vms/virt-enabling-disabling-vm-delete-protection.adoc#virt-enabling-disabling-vm-delete-protection-web_virt-enabling-disabling-vm-delete-protection[Enabling or disabling virtual machine delete protection by using the web console]
```

Comment:
This link doesn't work, and if this is an xref, it can't be in the module. You'll need to do the workaround where you add the xref in the additional resources section in the assembly below the module inclusion. Feel free to ping me if you need help with how to do that.